content/1.7/back-office/quick-access/add-edit-quick-access.md

```diff
@@ -7,9 +7,9 @@ weight: 1
 
 The following fields/toggles are displayed:
 
-- **Name**: this field is mandatory. When trying to save and if the field is empty, the following error message is displayed: "The field name is required in _default language_." The following tooltip is displayed: "Forbidden characters:<>;=#{}". When trying to save and if the field is invalid the following error message is displayed: "The name field is invalid". This field can be translated: When several languages are installed, a drop-down next to the field is displayed with default language selected. It allows choosing in which language the field is displayed and customizing it according to the language. Disabled languages are displayed in the drop-down. 
- - **URL**: this field is mandatory. When trying to save and if the field is empty, the following error message is displayed: "The link field is required.".
-The following tooltip is displayed: "If it's an URL that comes from your back office, you MUST remove the security token.".
+- **Name**: this field is mandatory for the default language. When trying to save and if the field is empty, the following error message is displayed: "The field name is required in _default language_." The following tooltip is displayed: "The maximum allowed characters is 32.". This field can be translated: When several languages are installed, a drop-down next to the field is displayed with default language selected. It allows choosing in which language the field is displayed and customizing it according to the language. Disabled languages are displayed in the drop-down. The field has a character limit of 32, it does not allow to enter more of them.
```

Comment:
@Saimis777 @marwachelly 

I don't see the tooltip "The maximum allowed characters is 32."
This is the tooltip displayed on my side:
<img width="1142" alt="Capture d’écran 2022-01-25 à 15 37 00" src="https://user-images.githubusercontent.com/13449658/150997447-65a8b36b-59b0-424f-b3dc-130eef641815.png">

Reply:
The existing tooltip should not be removed. IMO We can specify in the spec the number of maximum characters allowed but the tooltip isn't the appropriate place for this I think.

Reply:
Ok, I've changed back to the original tooltip.